openapi.yaml

```diff
@@ -3859,6 +3859,19 @@ paths:
                   $ref: '#/components/schemas/DatabaseMySQLRequest/properties/allow_list'
                 updates:
                   $ref: '#/components/schemas/DatabaseMySQL/properties/updates'
+                type:
+                  type: string
+                  description: |
+                    Request re-sizing of your cluster to a higher plan by specifying a larger [Linode Type](/docs/api/linode-types/#types-list).
+
+                    * Needs to be a higher plan than your current Linode Type.
+
+                    * You can't resize from a shared CPU Linode Type to a dedicated CPU, and vice-versa.
```

Comment:
This is not accurate -- switching between shared/dedicated _is_ supported.  (One slightly esoteric caveat: switching from a dedicated plan to a shared plan with the same specs is not currently supported, but that's also not intentional.  Switching in the opposite direction is supported, though.)

Reply:
Thanks, @akasklock ... Just to confirm: By "same specs," are you saying that you can't switch from a 1GB dedicated Linode to a 1GB shared Linode? This wouldn't be "upscaling" then, and it's covered by the point, "Needs to be a higher plan..."

So, I should just drop this whole point, right?

Reply:
I think that's mostly correct, albeit with a couple caveats.

* Switching from, e.g., 1GB shared to 1GB dedicated is supported.  If the median reader would understand that to be "upscaling", then I think dropping this bullet is sufficient.
* We're hoping to remove the limitation on like-to-like switching from dedicated to shared very soon (I'd characterize it as a bug).  I don't know if that has any implications for this PR, though -- perhaps it's just guidance that we're likely to request another small tweak to this verbiage in the near future.

Reply:
Thanks a lot for the explanation. I've removed the point for now. We can tweak these docs later when the fix is in place.